crates/test-utils/src/messages.rs

```diff
@@ -326,6 +331,7 @@ pub fn make_publish_basics_transaction(gas_object: ObjectRef) -> VerifiedTransac
         sender,
         gas_object,
         all_module_bytes,
+        vec![],
```

Comment:
This package depends on `Sui` at least -- should be able to use the `package_dependencie` helper here to figure that out.